/* Raven 2 Control - Control software for the Raven II robot
 * Copyright (C) 2005-2012  H. Hawkeye King, Blake Hannaford, and the University of Washington
 *BioRobotics Laboratory
 *
 * This file is part of Raven 2 Control.
 *
 * Raven 2 Control is free software: you can redistribute it and/or modify
 * it under the terms of the GNU Lesser General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
 *
 * Raven 2 Control is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU Lesser General Public License for more details.
 *
 * You should have received a copy of the GNU Lesser General Public License
 * along with Raven 2 Control.  If not, see <http://www.gnu.org/licenses/>.
 */
#ifndef RECONFIGURE_H_
#define RECONFIGURE_H_
#include <dynamic_reconfigure/server.h>
#include <raven_2/MyStuffConfig.h>

struct offsets {
    double shoulder_off;
    double elbow_off;
    double insertion_off;
    double roll_off;
    double wrist_off;
    double grasp1_off;
    double grasp2_off;
};

void reconfigure_callback(raven_2::MyStuffConfig &config, uint32_t level);
